local.tf

```diff
@@ -49,6 +49,7 @@ locals {
     bootstrap_extra_args          = ""                         # Extra arguments passed to the bootstrap.sh script from the EKS AMI.
     additional_userdata           = ""                         # userdata to append to the default userdata.
     ebs_optimized                 = true                       # sets whether to use ebs optimization on supported types.
+    cpu_credits                   = "standard"                 # T2/T3 unlimited mode, can be 'standard' or 'unlimited'. Used 'standard' mode as default to avoid paying higher costs
```

Comment:
Use `unlimited` as default is better because T3 instances start in Unlimited mode by default

Reply:
Hmm to quote the TF doc:

> T3 instances are launched as unlimited by default. T2 instances are launched as standard by default.

So not sure what default we want. Maybe stick `standard` as it's a safer option in terms of costs.

Reply:
@a-shink This variable should move to line 76  which is Settings for launch templates

Reply:
@nauxliu Done 00faa0e

Reply:
@nauxliu Is something blocking approving the PR? I don't change defaults because @max-rocket-internet thinks in similar way. Many peoples thinks about 'unlimited' default as money trap.